Allow "make install" to properly deliver the augeas lenses files, if configured.
Fix posted upstream so this patch should not be needed after nut-2.7.5 or so.

diff -Naur nut-2.7.4/configure.ac nut-2.7.4-auginst/configure.ac
--- nut-2.7.4/configure.ac	2016-05-02 19:07:03.924039136 +0200
+++ nut-2.7.4-auginst/configure.ac	2016-05-03 10:14:10.192735403 +0200
@@ -67,6 +67,10 @@
 driverexecdir='${exec_prefix}/bin'
 htmldir='${prefix}/html'
 pkgconfigdir='${libdir}/pkgconfig'
+auglensdir='/usr/share/augeas/lenses'
+if test ! -d "${auglensdir}"; then
+   auglensdir=''
+fi
 hotplugdir='/etc/hotplug'
 if test ! -d "${hotplugdir}"; then
    hotplugdir=''
@@ -1154,6 +1158,34 @@
 fi
 AM_CONDITIONAL(HAVE_SYSTEMD, test "$systemdsystemunitdir" != "")
 
+AC_MSG_CHECKING(whether to install Augeas configuration-management lenses)
+AC_ARG_WITH(augeas-lenses-dir,
+	AS_HELP_STRING([--with-augeas-lenses-dir=PATH], [where to install Augeas configuration-management lenses (/usr/share/augeas/lenses)]),
+[
+	case "${withval}" in
+	yes)
+		if test -z "${auglensdir}"; then
+			AC_MSG_RESULT(no)
+			AC_MSG_ERROR(["augeas lenses directory requested but not found"])
+		fi
+		;;
+	auto)
+		;;
+	no)
+		auglensdir=""
+		;;
+	*)
+		auglensdir="${withval}"
+		;;
+	esac
+], [])
+if test -n "${auglensdir}"; then
+	AC_MSG_RESULT(using ${auglensdir})
+else
+	AC_MSG_RESULT(no)
+fi
+AM_CONDITIONAL(WITH_AUGLENS, test -n "${auglensdir}")
+
 AC_MSG_CHECKING(whether to install hotplug rules)
 AC_ARG_WITH(hotplug-dir,
 	AS_HELP_STRING([--with-hotplug-dir=PATH], [where to install hotplug rules (/etc/hotplug)]),
@@ -1343,6 +1375,7 @@
 AC_SUBST(pkgconfigdir)
 AC_SUBST(systemdsystemunitdir)
 AC_SUBST(systemdsystemshutdowndir)
+AC_SUBST(auglensdir)
 AC_SUBST(hotplugdir)
 AC_SUBST(udevdir)
 
diff -Naur nut-2.7.4/Makefile.am nut-2.7.4-auginst/Makefile.am
--- nut-2.7.4/Makefile.am	2016-03-09 14:24:11.000000000 +0100
+++ nut-2.7.4-auginst/Makefile.am	2016-05-03 10:17:53.804392904 +0200
@@ -14,14 +14,15 @@
 # ----------------------------------------------------------------------
 # flags to pass to ./configure when calling "make distcheck" and "make
 # distcheck-light". Try to check as many features as possible! Also
-# need to give hotplug-dir and udev-dir, so that staged install does
-# not fail.
+# need to give augeas-lenses-dir, hotplug-dir and udev-dir, so that
+# staged install does not fail.
 
 DISTCHECK_FLAGS = --with-all --with-ssl --with-doc=auto
 DISTCHECK_LIGHT_FLAGS = --with-all=auto --with-ssl=auto --with-doc=auto
 
 DISTCHECK_CONFIGURE_FLAGS = ${DISTCHECK_FLAGS}		\
  --with-systemdsystemunitdir='$${prefix}/lib/systemd/system' \
+ --with-augeas-lenses-dir='$${prefix}/usr/share/augeas/lenses'		\
  --with-hotplug-dir='$${prefix}/etc/hotplug'		\
  --with-udev-dir='$${prefix}/etc/udev'			\
  --with-devd-dir='$${prefix}/etc/devd'
diff -Naur nut-2.7.4/scripts/augeas/Makefile.am nut-2.7.4-auginst/scripts/augeas/Makefile.am
--- nut-2.7.4/scripts/augeas/Makefile.am	2016-05-03 10:22:23.738243382 +0200
+++ nut-2.7.4-auginst/scripts/augeas/Makefile.am	2016-03-09 14:24:11.000000000 +0100
@@ -14,3 +14,12 @@
 		echo "Skipping Augeas ups.conf lens regeneration."; \
 		echo "----------------------------------------------------------------------"; \
 	fi
+
+if WITH_AUGLENS
+# Now "make install" should cover delivery of Augeas lenses...
+# The "auglensdir" value should be set up by configure
+# The *.aug files are generated by rule above or by autogen.sh and/or configure
+ auglens_DATA = \
+    nuthostsconf.aug  nutupsconf.aug   nutupsdusers.aug   nutupsschedconf.aug \
+    nutnutconf.aug    nutupsdconf.aug  nutupsmonconf.aug  nutupssetconf.aug
+endif
